pkg/component/watch.go

```diff
@@ -134,27 +134,29 @@ func WatchAndPush(client *occlient.Client, componentName string, applicationName
 		for {
 			select {
 			case event := <-watcher.Events:
+				isIgnoreEvent := false
 				changeLock.Lock()
 				glog.V(4).Infof("filesystem watch event: %s", event)
 
 				if !(event.Op&fsnotify.Remove == fsnotify.Remove || event.Op&fsnotify.Rename == fsnotify.Rename) {
 					stat, err := os.Lstat(event.Name)
 					if err != nil {
 						glog.Errorf("Failed getting details of the changed file %s", event.Name)
-						watchError = errors.Wrap(err, "unable to watch changes")
+						//watchError = errors.Wrap(err, "unable to watch changes")
 					}
 					// Some of the editors generate temporary buffer files during update to the file and deletes it soon after exiting from the editor
 					// Avoid pushing such buffer files
 					if stat == nil {
-						break
+						glog.V(4).Infof("Ignoring event for temp file %s", event.Name)
+						isIgnoreEvent = true
```

Comment:
How do you know that it is a temporary file?

Reply:
Because, the file was almost quickly(within interval between the event to handling the event) deleted . But I think I'll change it to say ignoring event for the now non-existant file

Reply:
to be more relaistic than making assumptions

Reply:
Marking the conversation resolved as the comments have been incoporated